UI/Panels/InputConfigPanel.cs

```diff
@@ -245,14 +254,40 @@ private void duplicateInputsRowToolStripMenuItem_Click(object sender, EventArgs
             }
         }
 
+        private bool IsSortingActive()
+        {
+            return inputsDataGridView.SortOrder != SortOrder.None;
+        }
+
+        private void inputsDataGridView_Sorted(object sender, EventArgs e)
+        {
+            if (IsSortingActive())
+            {
+                string name = inputsDataGridView.SortedColumn.Name;
+                // It is always (Ascending) -> (Descending) -> (Ascending)
+                // Reset sorting after previous Descending
+                if (LastSortingColumnName == name && LastSortingOrder == SortOrder.Descending)
```

Comment:
If we move this into its own method, then we can also call it when opening / creating a different file.

I really think it should be reset when dealing with a new file.

Reply:
Done.